bigbluebutton-html5/imports/ui/components/lock-viewers/context/context.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export function DataStruct() {
```

Comment:
Would it not make more sense for this to be named "LockStruct" or anything other than the super generic "DataStruct"? What happens when we want to have another context?